src/store/concepts/session/__tests__/actions.spec.js

```diff
@@ -0,0 +1,54 @@
+import {
+  loginUserSuccess,
+  loginUser,
+  logoutUser,
+  logoutUserSuccess
+} from '../actions'
+import {
+  USER_LOGIN_SUCCESS,
+  USER_LOGOUT,
+  USER_LOGIN_SUBMIT,
+  USER_LOGOUT_SUCCESS
+} from '../types'
+
+describe('session actions', () => {
+  describe('loginUserSuccess()', () => {
+    it('returns correct result', () => {
+      expect(loginUserSuccess(true)).toStrictEqual({
```

Comment:
do we have cases when we need to pass `false` to the `loginUserSuccess` action?

Reply:
I dont know, why I call with true argument:) I remove it